dao/src/main/java/com/garret/movies/dao/entity/Actor.java

```diff
@@ -0,0 +1,16 @@
+package com.garret.movies.dao.entity;
+
+import lombok.Data;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+@Table(name = "actor")
+public class Actor {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
why object? Id can be NULL?

Reply:
@mmarusich Yes. For new entity until it's not persisted id is null ))